backends/dpdk/constants.h

```diff
@@ -25,6 +25,10 @@ const unsigned action_handle_prefix = 0x00020000;
 const unsigned dpdk_default_table_size = 65536;
 #define DPDK_MAX_SHIFT_AMOUNT 64
 
+// Maximum number of configurable timeout values
+const unsigned dpdk_learner_max_configurable_timeout_values = 8;
+const unsigned default_learner_table_size = 0x10000;
```

Comment:
add macro for the constant

Reply:
How is using macro better than using const ?

Reply:
I think using const is better because it is type checked. Macros are not.